### DISCUSSION TITLE
일급컬렉션에서 추가적인 필드 선언을 허용하지 않는 이유

### DISCUSSION THREAD

**Discussion by 2yujeong** [woowa-Effective-Java-Study/effective-java-study#6] [General]
토론해주세요

**Comment by seong-wooo** [comment]
컬렉션을 집중관리하는 로직만을 담아놓기 위해 만드는 것이 일급컬렉션이라고 생각이 드네욥

**Comment by OzRagwort** [comment]
추가적인 필드를 선언하지 않아도 뭐 되긴한다.

일급 컬렉션은 다른 필드 선언을 허용하지 않는다기보다 선언해도 되지만 선언하면 그냥 객체가 되는 것이다.

배카라 : 배카라의 답변은 예리했다.
